### PR TITLE
NO-REF: Add export for type and fix warnings

### PR DESCRIPTION
**Pull request by jackiequach**
- Add export for ReadiumLink and WebpubManifest types to be used in consuming apps
- Fix warnings from browser for deprecated functions

### CODE REVIEW

**Review by Toxiapo — APPROVED** [review]

File: src/ui/nypl-base-theme/foundations/breakpoints.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // theme.js
-import { createBreakpoints } from '@chakra-ui/theme-tools';
-
-export default createBreakpoints({
```
Comment (kristojorg) [inline src/ui/nypl-base-theme/foundations/breakpoints.ts:4]: Just wondering, why remove the `createBreakpoints` function? It's only really there to aid in type checking

File: tests/jest.setup.ts [hunk 2]
```diff
@@ -1,2 +1,4 @@
 import '@testing-library/jest-dom';
 import 'jest-axe/extend-expect';
+
+jest.setTimeout(15000);
```
Comment (kristojorg) [inline tests/jest.setup.ts]: Does this need to be here?

**Review by kristojorg — COMMENTED** [review]
Just two questions, but otherwise looks great!

File: src/ui/nypl-base-theme/foundations/breakpoints.ts (hunk 1 above)
Comment (jackiequach) [reply src/ui/nypl-base-theme/foundations/breakpoints.ts:4]: There was a warning that this is being deprecated by Chakra soon and to just pass the breakpoints object to the theme.

File: tests/jest.setup.ts (hunk 2 above)
Comment (jackiequach) [reply tests/jest.setup.ts]: I moved the timeout to the specific tests (accessibility checks).

File: src/ui/nypl-base-theme/foundations/breakpoints.ts (hunk 1 above)
Comment (kristojorg) [reply src/ui/nypl-base-theme/foundations/breakpoints.ts:4]: Sweet thanks for the explanation!

**Review by kristojorg — APPROVED** [review]
Looks good!

**Pull request merged**